src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs

```diff
@@ -0,0 +1,77 @@
+﻿using SFA.DAS.Admin.Aan.Application.Constants;
+
+namespace SFA.DAS.Admin.Aan.Web.Models.ManageEvent;
+
+public class CheckYourAnswersViewModel : ManageEventViewModelBase
+{
+    public string? PreviewLink { get; set; }
+    public bool HasSeenPreview { get; set; }
+
+    public EventFormat? EventFormat { get; set; }
+
+    public string? EventTitle { get; set; }
+    public string? EventType { get; set; }
+    public string? EventRegion { get; set; }
+
+    public string? EventOutline { get; set; }
+    public string? EventSummary { get; set; }
+    public bool? HasGuestSpeakers { get; set; }
+    public List<GuestSpeaker> GuestSpeakers { get; set; } = new List<GuestSpeaker>();
+
+    public DateTime? Start { get; set; }
+    public DateTime? End { get; set; }
+
+    public string? EventLocation { get; set; }
+    public string? OnlineEventLink { get; set; }
+
+    public string? SchoolName { get; set; }
+
+    public bool? IsAtSchool { get; set; }
+
+    public string? OrganiserName { get; set; }
+    public string? OrganiserEmail { get; set; }
+
+    public int? NumberOfAttendees { get; set; }
+
+    public bool ShowLocation =>
+        EventFormat is Application.Constants.EventFormat.InPerson or Application.Constants.EventFormat.Hybrid;
```

Comment:
Application.Constants is already in the namespace, is there a collision?

Reply:
yes, there is a field in the viewModel called EventFormat